src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -1469,7 +1469,7 @@ public Worker build() {
             }
 
             if (shardPrioritization == null) {
-                shardPrioritization = new ParentsFirstShardPrioritization(1);
```

Comment:
When did this change?

Reply:
This wasn't ever changed, this is just out of sync with the current [default](https://github.com/ashwing/amazon-kinesis-client/blob/0afe3235252f43dfcfcaf75972777ae271c3a427/src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisClientLibConfiguration.java#L209). Basically if the config isn't specified and you use the worker builder, it uses the ParentFirstShardPrioritization instead, even though the default is supposed to be NoOpShardPrioritization. This causes some weird issue where if there are SHARD_END shards it doesn't ever get scheduled by the Worker since they're assigned a priority of 0.